components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/registration.jsp

```diff
@@ -14,54 +14,87 @@
 * KIND, either express or implied. See the License for the
 * specific language governing permissions and limitations
 * under the License.
--->
+*/ %>
 <%@ page
         import="org.wso2.carbon.identity.application.authentication.endpoint.util.UserRegistrationAdminServiceClient" %>
-<%@ page import="java.util.Map" %>
-<%@ page import="java.util.HashMap" %>
 <%@ page import="org.owasp.encoder.Encode" %>
-<%@ page import="org.wso2.carbon.context.CarbonContext" %>
+<%@ page import="org.wso2.carbon.identity.user.registration.stub.dto.UserFieldDTO" %>
+<%@ page import="java.util.List" %>
+<%@ page import="org.apache.commons.lang.StringUtils" %>
 
 <%
     String forwardTo;
-    int errorCode = 0;
     try {
+
         UserRegistrationAdminServiceClient registrationClient = new UserRegistrationAdminServiceClient();
-        boolean isExistingUser = CarbonContext.getThreadLocalCarbonContext().getUserRealm().
-                getUserStoreManager().isExistingUser(request.getParameter("reg-username"));
-        if (isExistingUser){
-            errorCode = 1;
-            throw new Exception();
+        boolean isExistingUser = registrationClient.isUserExist(request.getParameter("reg_username"));
```

Comment:
We can assign request.getParameter("reg_username") to a variable and re-use.I noticed those parameter values  have re-use in different places